### DISCUSSION TITLE
Welcome to djangocms-bootstrap4 Discussions!

### DISCUSSION THREAD

**Discussion by NicolaiRidani** [django-cms/djangocms-bootstrap4#146] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fsbraun** [comment]
Thanks Nicolai for opening the forum! I am working on migrating from Bootstrap 4 to Bootstrap 5. Please see #148 for a proposal on how to (a) get this migration done and (b) how to decouple this app from future upgrades in Bootstrap - even maybe offering support for other css frameworks in the future. Love to hear your thoghts!
